### ISSUE TITLE
Ready for Plone 6 final?

### ISSUE THREAD

**Issue by mauritsvanrees** [plone/bobtemplates.plone#534]
Is there anything that needs to happen before we can truly say `bobtemplates.plone` can be used for Plone 6 projects? Any nasty open issues?

Python 3.11 support would be nice to have, Plone 6.0.0 RC1 supports this.

**Comment by mauritsvanrees** [comment]
We can close this. :-)
